From Frugalware:

* disable library destructor
  it triggers a glibc bug where elf dynamically loaded libraries
  are freed before the library constructor is ever called

--- a/src/vdpau_wrapper.c	2013-01-24 18:55:06.000000000 -0600
+++ b/src/vdpau_wrapper.c	2013-07-15 20:14:40.221679876 -0500
@@ -47,7 +47,7 @@
 
 #if defined(__GNUC__)
 
-static void _vdp_close_driver(void) __attribute__((destructor));
+static void _vdp_close_driver(void);
 
 #endif
 
